cli/docs/cli.json

```diff
@@ -4566,7 +4633,7 @@
                   "subcommands": [
                     {
                       "name": "add",
```

Comment:
The subcommand is still named `add`, even though the code will allow updates when calling `add` on a peer that already exists.

Reply:
Fixed